src/components/Layout/Sidebar/SidebarLink.tsx

```diff
@@ -71,12 +71,17 @@ export function SidebarLink({
         }
       )}>
       {/* This here needs to be refactored ofc */}
-      <span
-        className={cn({
-          'text-gray-400 dark:text-gray-500': wip,
-        })}>
-        {title}
-      </span>
+      <div>
+        {title}{' '}
+        {canary && (
+          <span
+            title="This feature is available in the latest React Canary"
+            className="ml-2 border border-purple-40 dark:border-purple-50 rounded-full pt-0 pb-0.5 px-2 text-sm text-purple-40 dark:text-purple-50 leading-3">
+            canary
```

Comment:
I prefer the look of it capitalized, but I get that that's super subjective haha. Looks great regardless! Very easy to scan and tell. 

I like the purple but was wondering if the colors should be more "traditional"? I think yellow is typically associated with canary (yellow Chrome icon, yellow Meta app icons, etc).

Reply:
Capitalizing it makes the pill feel bigger I think.

Good call on the yellow. Here's yellow in our current color scheme, which looks more brown than yellow:

<img width="1710" alt="Screenshot 2023-07-21 at 12 39 40 PM" src="https://github.com/reactjs/react.dev/assets/2440089/a3d09622-e649-46d9-adb3-06194014cae5">

<img width="1710" alt="Screenshot 2023-07-21 at 12 39 30 PM" src="https://github.com/reactjs/react.dev/assets/2440089/ca2e84e8-e66c-41ce-9521-c6630d13da4b">

Reply:
I think you might have pasted the wrong screenshot, I still see purple

Reply:
@poteto oops, fixed!

Reply:
Ok I pushed the yellow